app/controllers/orgs/rosters_controller.rb

```diff
@@ -38,9 +38,10 @@ class RostersController < Orgs::Controller
     before_action :google_classroom_ensure_no_roster, only: %i[
       select_google_classroom
     ]
-
     helper_method :current_roster, :unlinked_users, :authorize_google_classroom
 
+    depends_on :lti
```

Comment:
In the interest of keeping `rosters_controller` more or less untouched and decrease mental overhead, I used the awesome `depends_on:` controller helper we use in a few other large controllers -- it essentially splits up the controller, kind of like a mixin, and allows us to compartmentalize controller logic